### DISCUSSION TITLE
通过 Popclip App 将选中文字发送到 flomo

### DISCUSSION THREAD

**Discussion by lightory** [flomoapp/3rd-party-tools#6] [分享创造]
https://blog.flomoapp.com/55.html

**Comment by geekdada** [comment]
原项目在这 https://github.com/geekdada/PopClip-Extensions

**Comment by levie-vans** [comment]
我之前也写了一个，写完才发现有人已经写出来了。功能也差不多
https://github.com/levie-vans/Flomo_PopClip_Extension/
